src/test/kotlin/blackjack/domain/CardTest.kt

```diff
@@ -0,0 +1,45 @@
+package blackjack.domain
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+
+internal class CardTest : FunSpec({
+    context("Denomination이 ACE 이면 isAce()가 true이다.") {
+        data class CardTestData(
+            val cardSuit: CardSuit,
+            val denomination: Denomination,
+        )
+
+        withData(
+            ts = listOf(
+                CardTestData(cardSuit = CardSuit.HEART, denomination = Denomination.ACE),
+                CardTestData(cardSuit = CardSuit.CLOVER, denomination = Denomination.ACE),
+            )
```

Comment:
```suggestion
            ts = listOf(
               CardSuit.HEART to Denomination.ACE,
               CardSuit.CLOVER to Denomination.ACE,
            )
```

불필요한 CardTestData를 선언하지 않아도 될거같아요!